doc/md/getting-started/examples.md

```diff
@@ -0,0 +1,108 @@
+
+- [Reversi](https://github.com/ninegua/reversi): A multiplayer Reversi game.
+
+- [Revo](https://github.com/DepartureLabsIC/revo): A drawing game.
+
+- [SecureGuard Escrow](https://github.com/bix-tech/secure-guard-escrow): ICP-based escrow platform.
+
+### Tokens & DeFi
+
+- [Appic](https://github.com/Appic-Solutions/Auto_Investment): Automatically by and sell tokens at a specific interval or time.
+
+- [ICDex](https://github.com/iclighthouse/ICDex): An on-chain orderbook decentralized exchange written in Motoko.
+
+- [icrc7_motoko](https://github.com/noku-team/icrc7_motoko) and - [icrc7-mo](https://github.com/PanIndustrial-Org/icrc7.mo): Motoko implementations of the ICRC-7 NFT standard.
+
+- [icrc-nft-mo](https://github.com/PanIndustrial-Org/icrc_nft.mo): An example NFT that combines `icrc7-mo`, `icrc30-mo`, and `icrc3-mo`.
+
+- [ic-token](https://github.com/rocklabs-io/ic-token) and [motoko-token](https://github.com/enzoh/motoko-token): Motoko implementations of an ERC-20 token.
+
+- [Itoka music NFT](https://github.com/Itoka-DAO/itoka-music-nft-standard): A standard for music NFTs.
+
+- [Motoko Bitcoin](https://github.com/tgalal/motoko-bitcoin): A Motoko Bitcoin integration.
+
+- [Staking pool](https://github.com/ICTO-Labs/staking-pool): A staking pool project written in Motoko.
+
+- [Token faucet](https://github.com/rocklabs-io/token-faucet): A token faucet for [ic-token](https://github.com/rocklabs-io/ic-token).
+
+- [Token minter sample](https://github.com/icpipeline-framework/mbt): A sample app that mints tokens.
+
+### Writing Motoko
+
+- [Blocks](https://blocks-editor.github.io/blocks/): A Motoko smart contract editor.
+
+- [JSON for Motoko](https://github.com/aviate-labs/json.mo): Motoko JSON library.
+
+- [motoko-library-template](https://github.com/kritzcreek/motoko-library-template): Template for creating Motoko libraries.
+
+- [Motoko Pilot](https://d7dm6-sqaaa-aaaag-qcgma-cai.icp0.io/): An AI tool for writing Motoko.
+
+- [Motoko starter](https://github.com/motoko-bootcamp/motoko-starter): Basic starter template for Motoko projects.
+
+- [Vite + React + Motoko template](https://github.com/rvanasa/vite-react-motoko): Boilerplate template for Vite, React, and Motoko.
+
+- [Visual Studio Code Motoko extension](https://github.com/dfinity/vscode-motoko): A VS Code extension for Motoko.
+
+
+## Education resources
+
+- [Awesome Motoko](https://github.com/ic123-xyz/awesome-motoko): A list of Motoko projects, tools, and resources.
+
+- [DFINITY Examples](https://github.com/dfinity/examples/tree/master/motoko): Simple Motoko sample projects and templates.
+
+- [ICP developer journey](https://internetcomputer.org/docs/current/tutorials/developer-journey/): A 36-part series focused on Motoko development for ICP canisters.
+
+- [Motoko book](https://github.com/Web3NL/motoko-book): An educational book on Motoko.
+
+- [Motoko Bootcamp](https://github.com/motoko-bootcamp/education): Lessons and lectures on Motoko development.
```

Comment:
Not sure whether we should perhaps add https://github.com/luc-blaeser/auction, the Motoko CySep summer school workshop at KTH Sweden.

Reply:
Very nice collection!

Reply:
Added the auction example :)